src/common/utils/general/general.ts

```diff
@@ -117,6 +117,10 @@ function log10(n: number) {
   return Math.log(n) * Math.LOG10E;
 }
 
+export function integerDivision(x: number, y: number): number {
```

Comment:
I would use more specific name like divideAndFloor this function does not just divide integers ...

Reply:
Argumentum ad verecundiam: http://mathworld.wolfram.com/IntegerDivision.html